bellboy/sensors.py

```diff
@@ -0,0 +1,145 @@
+# Sensor classes
+# Sensors include Camera, Ultrasonicm Mic....
+import logging
+import time
+from threading import Thread
+
+import picamera
+import RPi.GPIO as GPIO
+
+logger = logging.getLogger(__name__)
+
+
+class AbstractSensor():
+
+    # constructor.
+    # creates private variables for:
+    #   the sensor's data buffer,
+    #   the polling period, and
+    #   the sensor's polling thread
+    # @param poll_period_us time inbetween sensor reads, in microseconds
+    def __init__(self, poll_period_us):
```

Comment:
Codacy found an issue: [Missing docstring in __init__ (D107)](https://app.codacy.com/gh/Bellboy-Capstone/System/pullRequest?prid=6272175)